MAUI/Autocomplete/Header-and-Footer.md

```diff
@@ -0,0 +1,211 @@
+                 Padding="30">
+        <autocomplete:SfAutocomplete HeightRequest="40"
+                                     ShowDropDownHeaderView ="True"
+                                     ItemsSource="{Binding SocialMedias}"
+                                     DisplayMemberPath="Name"
+                                     DropDownHeaderViewHeight="50">
+            <autocomplete:SfAutocomplete.DropDownHeaderView>
+                <StackLayout BackgroundColor="#f0f0f0" >
+                    <Label  x:Name="SearchLabel" 
+                            FontSize="20" 
+                            VerticalTextAlignment="Center" 
+                            HorizontalOptions="Center" 
+                            VerticalOptions="Center" 
+                            TextColor="#006bcd"   />
+                </StackLayout>
+            </autocomplete:SfAutocomplete.DropDownHeaderView>
+        </autocomplete:SfAutocomplete>
+    </StackLayout>
+</ContentPage>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+using Syncfusion.Maui.Inputs;
+using System.Collections.Generic;
+
+namespace AutocompleteSample
+{
+    public partial class MainPage : ContentPage
+    {
+        Label SearchLabel;
+        public MainPage()
+        {
+            InitializeComponent();
+            StackLayout stackLayout = new StackLayout
+            {
+                VerticalOptions = LayoutOptions.Start,
+                HorizontalOptions = LayoutOptions.Start,
+                Padding = new Thickness(30)
+            };
+
+            SfAutocomplete autoComplete = new SfAutocomplete
+            {
+                HeightRequest = 40,
+                ShowDropDownHeaderView = true,
+                ItemsSource = new List<string> { "Facebook", "Twitter", "Instagram", "LinkedIn" },
+                DropDownHeaderViewHeight = 50
+            };
+
+            StackLayout dropDownHeaderView = new StackLayout
+            {
+                BackgroundColor = Color.FromHex("#f0f0f0")
+            };
+
+            Label searchLabel = new Label
+            {
+                FontSize = 20,
+                VerticalTextAlignment = TextAlignment.Center,
+                HorizontalOptions = LayoutOptions.Center,
+                VerticalOptions = LayoutOptions.Center,
+                TextColor = Color.FromHex("#006bcd")
+            };
+            dropDownHeaderView.Children.Add(searchLabel);
+            autoComplete.DropDownHeaderView = dropDownHeaderView;
+            stackLayout.Children.Add(autoComplete);
+            Content = stackLayout;
+        }
+    }
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Header Image](Images/HeaderFooter/headertemplate.png)
+
+## Footer Content
+
+We can provide Footer Content at the bottom of the Autocomplete's dropdown. [`DropDownFooterView`] property is used to set the content of the footer.The height of the Header in the [`SfAutocomplete`] can be adjusted by the property [`DropDownFooterViewHeight`].
```

Comment:
The height of the Header in the [`SfAutocomplete`] can be adjusted by the property
---
Header -> wrong word

Reply:
Changed header to Footer.